r/NEWS.md

```diff
@@ -79,6 +79,7 @@ well as all users who submitted github issues to help identify and fix bugs.
   graph-based clustering
 - Add `checksum()`, which allows for calculating an MD5 checksum of a matrix contents. Thanks to @brgrew for submitting this improvement in pull request #83
 - `write_insertion_bedgraph()` allows exporting pseudobulk insertion data to bedgraph format
+- Add `row_max()` and `col_max()` functions, which return the maximum value in each row or column of a matrix.
```

Comment:
This should go under the 0.2.1 section. (No guarantees we'll actually call that version 0.2.1, but we need some number there to get the documentation website to render properly)